client/src/pages/Home/Home.tsx

```diff
@@ -25,14 +69,20 @@ function Home(props: any) {
     }
   }, []);
 
+  useEffect(() => {
+    if (curGeoLocation) {
+      getStoresByLocation(curGeoLocation).then((res: any) => setStores(res));
+    }
+  }, [curGeoLocation]);
+
   useEffect(() => {
     if (userid) {
       getUserInfo(userid).then((res) => setCurUser(res));
     }
   }, [userid]);
 
   useEffect(() => {
-    setPlaceStore(testPlaces.map((place) => transformGMapPlaceToStore(place)));
+    setPlaceStores(testPlaces.map((place) => transformGMapPlaceToStore(place)));
```

Comment:
Add TODO here as well to update test places

Reply:
added TODO to remove when we have added Places API filtering functionality to backend /store/list call